packages/shared/src/fonts/gt-america.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: GT-America;
+  font-style: bold;
+  font-weight: 800;
+  font-display: swap;
+  src: url('GT-America-Black.woff2') format('woff2'),
+    url('GT-America-Black.woff') format('woff');
+}
+
+
```

Comment:
Do we really want to blow out our bundle size with a new font just for a couple of titles on the welcome screen? I don't think so 😄

Reply:
Not sure if I am understanding correctly: You do not mean that we should rather not use that font here? But as I have added multiple font-files of the GT-America which are currently not used, that produces too much overhead, right?
So, we should still use GT-America, but just the files we have in use. Kinda thought of setting it up and having a whole bunch of GT-America ready for future usage - but it might not be the most efficient way in regards of performance.

So, I will clean this up!